tox.ini

```diff
@@ -10,19 +10,12 @@ commands = nosetests {posargs}
 [testenv:lint]
 deps =
   pre-commit
```

Comment:
On thing I thought of and @asottile may know the answer for is:

When using `pre-commit` to run Black on CI (e.g., Travis CI), will it know to only do `black --check`? `black --check` merely verifies that running black would be a no-op rather than running it. That then helps confirm compliance on the developer side. (Hopefully I'm making sense)

Reply:
~usually the suggestion is to still have it reformat, but use `pre-commit run --all-files --show-diff-on-failure` -- for example: [from pytest-dev/pytest](https://github.com/pytest-dev/pytest/blob/a77c83a4c3908f5c82d3328ed8b636943959ecaa/tox.ini#L57)

Reply:
oops pressed enter too soon, pre-commit knows when a reformat happens to fail a run and the outputted diff can be directly applied via `git apply` + paste.  there's also some hint text with that set of arguments about local setup